### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by AweSamarth**
## Changes proposed

Added Hardhat and Foundry to the list of Open Source projects that one can contribute to.

## Screenshots
image

### CODE REVIEW

**Review by siddhant-khisty — APPROVED** [review]
Looks good to me. Thanks for your contribution.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
lgtm. Can you please resolve the merge conflict so we can merge it

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by AweSamarth** [comment]
Resolved merge conflict

File: Blockchain/Contribute/README.md [hunk 1]
```diff
@@ -40,3 +40,5 @@
 - [Web3j](https://github.com/web3j/web3j)
 - [Hardhat](https://github.com/NomicFoundation/hardhat)
 - [Foundry](https://github.com/foundry-rs/foundry)
+- [Remix Project](https://github.com/ethereum/remix-project)
+
```
Comment (kaiwalyakoparkar) [inline Blockchain/Contribute/README.md]: please remove this extra line

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**